pytorch_lightning/__init__.py

```diff
@@ -1,6 +1,6 @@
 """Root package info."""
 
-__version__ = '1.0.3'
+__version__ = '1.0.4a0'
```

Comment:
```suggestion
__version__ = '1.0.4rc0'
```
but not sure if we really need this since we have weekly scheduled releasing...
basically, this shall go always with updating Chlog :]